src/components/Card/readme.md

```diff
@@ -111,3 +111,143 @@ const iconContainerStyles = {
     </Card>
 </div>
 ```
+
+##### pricing card
+
+```js
+import React from 'react';
+import { Card, Button } from 'react-rainbow-components';
+import styled from 'styled-components';
+
+const Title = styled.h1`
+    font-family: 'Lato Light';
+    font-size: 32px;
+    font-weight: 100;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 1px;
+    text-align: center;
+    color: ${props => props.theme.rainbow.palette.text.main};
+`;
+
+const Subtitle = styled.h2`
+    font-family: 'Lato Light';
+    font-size: 16px;
+    font-weight: 700;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 0.5px;
+    text-align: center;
+    color: ${props => props.theme.rainbow.palette.text.header};
+`;
+
+const StyledCard = styled(Card)`
+    width: 240px;
+    height: 363px;
+`;
+
+const StyledH2 = styled.h2`
+    font-family: 'Lato Black';
+    font-size: 56px;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 56px;
+    letter-spacing: normal;
+    color: ${props => props.theme.rainbow.palette.brand.main};
+`;
+
+const StyledH3 = styled.h3`
+    font-family: Lato;
+    font-size: 20px;
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: ${props => props.theme.rainbow.palette.brand.main};
+`;
+
+const StyledP = styled.p`
+    font-family: 'Lato Light';
+    font-size: 15px;
+    font-weight: 700;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 0.5px;
+    text-align: center;
+    color: ${props => props.theme.rainbow.palette.text.main};
+`;
+
+const StyledB = styled.b`
+    font-family: 'Lato Bold';
+`;
+
+function PriceCard(props) {
+    const { packageType, packagePrice, projectsCount, members, contacts} = props;
+    return (
+        <StyledCard
+            className="rainbow-flex rainbow-flex_column rainbow-align_center rainbow-justify_space-around"
+        >
+            <Subtitle
+                style={{
```

Comment:
We should follow the principle of separations of concerns, we should not mix the inside the jsx.
Since you are using a styled-component to style this component you don't need to use inline-styles